### PR TITLE
(WIP) Add an idempotency test to unit integration tests.

### PR DESCRIPTION
**Pull request by RandomNoun7**
Add an idempotency test to your tests. I'm not sure this is the right branch to put it in, but it's the only one explicitly testing related.

### CODE REVIEW

**Comment by RandomNoun7** [comment]
It's not much, but learning has occurred.

File: spec/acceptance/should_destroy_spec.rb [hunk 1]
```diff
@@ -0,0 +1,44 @@
+require 'spec_helper_acceptance'
+
+host = find_only_one("default")
+
+describe "Should detroy a scheduled task", :node => host do
```
Comment (glennsarti) [inline spec/acceptance/should_destroy_spec.rb]: Typo

File: spec/integration/puppet_x/puppetlabs/scheduled_task/taskscheduler2_spec.rb [hunk 2]
```diff
@@ -146,6 +146,17 @@ def create_test_task(task_name = nil, task_compatiblity = PuppetX::PuppetLabs::S
       it 'should have the specified action arguments' do
         expect(subject.action(task_definition, 1).Arguments).to eq('/c exit 0')
       end
+
+      context 'it should behave like an idempotent resource' do
```
Comment (glennsarti) [inline spec/integration/puppet_x/puppetlabs/scheduled_task/taskscheduler2_spec.rb]: I'm not sure how I feel about an idempotency test here.  This is an integration test for the puppet_x classes which are called via the provider.  Puppet and the provider determine idempotency, not this class.

There is already an idempotency check in the `should_create` acceptance test.

Comment (Iristyle) [reply spec/integration/puppet_x/puppetlabs/scheduled_task/taskscheduler2_spec.rb]: Yeah, this probably belongs in provider.

**Pull request merged**